apis_core/utils/test_caching.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+
+from apis_core.utils import caching
+
+all_class_modules_and_names = {
+    "apis_metainfo": [
+        "Collection",
+        "RootObject",
+        "Source",
+        "Text",
+        "Uri",
+        "UriCandidate",
+    ],
+    "apis_vocabularies": [
+        "CollectionType",
+        "LabelType",
+        "TextType",
+        "VocabNames",
+        "VocabsBaseClass",
+        "VocabsUri",
+    ],
+    "apis_entities": ["TempEntityClass"],
+    "apis_relations": ["TempTriple", "Property", "Triple"],
+    "contenttypes": ["ContentType"],
+}
+
+
+class CachingTest(TestCase):
+    def test_get_all_class_modules_and_names(self):
+        expected = []
+        for key in all_class_modules_and_names:
+            for name in all_class_modules_and_names[key]:
+                expected.append((key, name))
+        res = caching.get_all_class_modules_and_names()
+        self.assertEqual(res, expected)
```

Comment:
As said, I'm unable to run the test at this point, what I'm wondering about is:

Wouldn't  `caching.get_all_class_modules_and_names()` include all `apis_ontology.models` as well?

(Or... is that the problem with my testing the test? Is this supposed to be limited to apis-core, i.e. not actually meant up to pick up the ontology models?)

Reply:
the tests are testing the `apis-core-rdf` project. there is no `apis_ontology.models` module anywhere.

Reply:
(In fact, the tests break when used with `apis_ontology`, because `apis_ontology` is not a real django app (yet))